docs/gadgets/trace/tcp.md

```diff
@@ -13,34 +13,35 @@ connect, accept and close events related to TCP connections.
 First, we need to create one pod:
 
 ```bash
-$ kubectl run busybox --image busybox:latest sleep inf
```

Comment:
Please update guides of other gadgets too.

Reply:
But the columns' names are different now: `DST.X` and `SRC.Y`.

Reply:
Ahhh yes. For `trace tcpdrop` and `trace tcp` I had to update them